lib/main.dart

```diff
@@ -17,10 +17,17 @@ import 'package:gallery/themes/gallery_theme_data.dart';
 
 void main() {
   GoogleFonts.config.allowHttp = false;
-  runApp(GalleryApp());
+  runApp(const GalleryApp(testMode: false));
```

Comment:
The default value is false so it only needs to be set to true in the integration tests